tgui/packages/tgui/interfaces/NtosMain.tsx

```diff
@@ -52,6 +55,24 @@ export const NtosMain = (props) => {
                   />
                 </Stack.Item>
               ))}
+              <Stack.Item
+                right = "0px"
+              >
+                <Button
+                  inline
+                  bold
+                  className = { (alert_style == 2) ? "alertBlink" : ""}
+                  right = "0px"
+                  textAlign= "center"
+                  verticalAlign = "center"
+                  textColor = { (alert_style == 0) ? alert_color : "#000000" }
+                  fontFamily = "Courier New"
+                  backgroundColor = { (alert_style == 0) ? "#0000000" : alert_color }
+                  tooltip = "The current alert level. Indicator becomes more intense when there is a threat, moreso if your department is responsible for handling it."
+                >
+                  { alert_name }
+                </Button>
```

Comment:
no spaces between prop name, equals and value
also tbh these are enough style props to justify a css class